portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,47 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact about myself to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ["I'm one of a triplet - and we are all females",
```

Comment:
Please use consistent quoting and indentation in each block of code (here the last line breaks both).

Having clean code makes it easier to read later.